common-pipeline/pipeline-flow/pipeline-flow-v3-schema.json

```diff
@@ -480,6 +480,10 @@
     "link_def": {
       "description": "Node link definition",
       "type": "object",
+      "id": {
```

Comment:
I think this belongs under the properties.  Right?

Reply:
Fixed